webservice/requirements.txt

```diff
@@ -1,48 +1,66 @@
+Active-Alchemy==1.0.0
 alembic==0.8.10
 appdirs==1.4.0
 APScheduler==3.3.1
+arrow==0.12.1
+asn1crypto==0.24.0
 backports-abc==0.5
+backports.functools-lru-cache==1.5
 bagit==1.6.3
 boto==2.46.1
+boto3==1.8.4
+botocore==1.11.4
 certifi==2017.1.23
 cffi==1.9.1
+chardet==3.0.4
 click==6.7
+configparser==3.5.0
 cryptography==2.1.4
 docutils==0.13.1
 elasticsearch==5.5.2
-elasticsearch-dsl>=5.0.0,<6.0.0
+elasticsearch-dsl==5.4.0
 enum34==1.1.6
+flake8==3.5.0
 Flask==0.12.2
 Flask-APScheduler==1.5.0
 Flask-Cors==3.0.2
 Flask-Elasticsearch==0.2.5
 Flask-Excel==0.0.5
+Flask-Login==0.4.0
```

Comment:
Why did the case of this change? It was `flask-login` before. Is the requirements.txt not case-sensitive (I don't know)?

Reply:
This was likely re-written before this commit: https://github.com/DataBiosphere/azul/pull/304/commits/2f8bd791aeed41298f365e67cbf29e2534be4602, where I installed needed packages. 

That likely also changed the order of the list. Perhaps `requirements.txt` was hand-edited before?

It seems that installation using pip is case-insensitive (https://stackoverflow.com/questions/26503509/is-pypi-case-sensitive)